src/darwin/Framework/CHIP/CHIPOperationalCredentialsDelegate.h

```diff
@@ -79,7 +79,8 @@ class CHIPOperationalCredentialsDelegate : public chip::Controller::OperationalC
     chip::NodeId mNextRequestedNodeId = 1;
     chip::FabricId mNextFabricId = 1;
     bool mNodeIdRequested = false;
-    bool mGenerateRootCert = false;
+    bool mForceRootCertRegeneration = false;
+    bool haveRootCert = false;
```

Comment:
Why does `haveRootCert` need to be a member variable?

Reply:
Removed and made it local.